docs/content/en/docs/yaml-crd-ref/evaluationdefinition.md

```diff
@@ -0,0 +1,99 @@
+---
+title: KeptnEvaluationDefinition
+description: Define an evaluation query
+weight: 20
+---
+
+A `KeptnEvaluationDefinition` defines evaluation tasks
+that can be run by the Keptn Lifecycle Toolkit
+as part of pre- and post-analysis phases of a workload or application.
+
+## Yaml Synopsis
+
+```yaml
+apiVersion: lifecycle.keptn.sh/v1alpha3
+kind: KeptnEvaluationDefinition
+metadata:
+  name: <evaluation-name>
+spec:
+  objectives:
+    - evaluationTarget: ">1"
+      keptnMetricRef:
+        name: available-cpus
+        namespace: some-namespace
+```
+
+## Fields
+
+* **apiVersion** -- API version being used.
+  Must be `v1alpha3` or later for this syntax.
+`
+* **kind** -- Resource type.
+   Must be set to `KeptnEvaluationDefinition`
+
+* **metadata**
+  * **name** -- Unique name of this evaluation
+    such as `pre-deploy-eval` or `post-deploy-resource-eval`.
+    Names must comply with the
+    [Kubernetes Object Names and IDs](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names)
+    specification.
+
+* **spec**
```

Comment:
I have mixed feelings about having this manually maintained. The APIs will change continuously and we need to keep this section aligned with the implementation. For the fields, we generate the documentation automatically in the API reference page: https://lifecycle.keptn.sh/docs/crd-ref/
I would rather keep it there so we don't have to maintain the fields in multiple places